preseed/finish.erb

```diff
@@ -26,17 +26,9 @@ oses:
 /bin/sed -i "s/^<%= @host.ip %>.*/<%= @host.ip %>\t<%= @host.shortname %>.<%= @host.domain %>\t<%= @host.shortname %>/g" /etc/hosts
 /bin/echo <%= @host.shortname %> > /etc/hostname
 <% end -%>
+
 <% if puppet_enabled %>
-cat > /etc/puppet/puppet.conf << EOF
-<%= snippet 'puppet.conf' %>
-EOF
-if [ -f "/etc/default/puppet" ]
-then
-/bin/sed -i 's/^START=no/START=yes/' /etc/default/puppet
-fi
-/bin/touch /etc/puppet/namespaceauth.conf
```

Comment:
This has been lost?  I think it might be for quite an old version, but perhaps worth keeping in.